### PR TITLE
Introduce CI based on github actions

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

**Comment by jfdenise** [comment]
This will need to get merged for next PR to activate it. My own run were green.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+jobs:
+  build:
+    name: ${{ matrix.os }}-jdk${{ matrix.java }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest ]
+        java: ['1.8', '11']
```
Comment (aloubyansky) [inline .github/workflows/ci.yml:36]: Do we still care about 1.8?

**Review by aloubyansky — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jfdenise) [reply .github/workflows/ci.yml:36]: We are still running JDK8 tests with wildfly. I will keep JDK8 for now and remove when no more needed.

Comment (jfdenise) [reply .github/workflows/ci.yml]: @aloubyansky I removed JDK1.8 and added source and binary level 1.8 in pom.

**Pull request merged**